classes/main.rb

```diff
@@ -4,6 +4,7 @@
 
 # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
- [ ] Same thing here as mentioned in the previous comment :point_up: Kindly do not disable any **rubocop** property.